app/views/mobile_phone/verify_create.html.haml

```diff
@@ -26,6 +26,11 @@
             oninput: "setCustomValidity('')"
         = button_tag "Verify"
 
+      - if flash[:errors]
+        %p.field-error
+          Oh dear, that doesn't seem to be right. We will send you a new 6
```

Comment:
This should appear immediately below the `<h1>`, in red, and in lieu of the "A verfication code was sent to ..." sentence.